File: Go4Lunch/app/src/main/java/com/openclassrooms/go4lunch/data/model/api/places/Geometry.java
package com.openclassrooms.go4lunch.data.model.api.places;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * Geometry Class from both Google Place Nearby Search Api and Place Details Api
 */
public class Geometry {

    @SerializedName("location")
    @Expose
    private Location location;

    public Geometry() {}

    public Location getLocation() {
        return location;
    }

    public void setLocation(Location location) {
        this.location = location;
    }
}

File: RealEstateManager/app/src/main/java/com/openclassrooms/realestatemanager/models/place/Geometry.kt
package com.openclassrooms.realestatemanager.models.place

import com.openclassrooms.realestatemanager.models.place.detail.Location
import com.openclassrooms.realestatemanager.models.place.detail.Viewport

data class Geometry (
    val location: Location,
    val viewport: Viewport
)